pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -0,0 +1,164 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package virtwrap
```

Comment:
Is it possible to perform the tests from `package virtwrap_test`?

Reply:
The interface I'm adding is not exported - since there's no need for anyone else to use it. 

... I can do that, but it'll require the interface to be exported, for no reason other than unit test it from a different package. I don't think that's good enough a reason to do so.

Do you ?

Reply:
Could you please point me to the place in the code where it is used?

Reply:
https://github.com/kubevirt/kubevirt/pull/9429/files#diff-a4d56356c36cdb90f1e866f87d3d84776fffe23bf5dbe1cc89d5b36dd94e4454R74